src/containers/Gurb/components/TextRecomendation.jsx

```diff
@@ -0,0 +1,22 @@
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+
+import { textHeader2, textBody1 } from '../gurbTheme'
+
+const TextRecomendation = ({ title, text }) => {
+  return (
+    <Box>
+      <Typography
+        // variant="h2"
+        sx={textHeader2}>
+        {title}
+      </Typography>
+      <Typography
+        // variant="body1"
+        sx={textBody1}>
+        {text}
+      </Typography>
+    </Box>
+  )
+}
+export default TextRecomendation
```

Comment:
The structure looks fine to me, but I think there are some components that could be reused from the generic components that are there now. For example the Next and Previous buttons.

Reply:
The new buttons (Next and Previous) are a bit different from what we have now... color, behavior, even icons. This is why we put it separately. In addition, these buttons might change in a near future, and we didn't want to spend time adapting everything before having the final version.